### ISSUE TITLE
Make grid control in style panel accessible

### ISSUE THREAD

**Issue by kof** [webstudio-is/webstudio#357]
Currently when you tab into it - layout starts shifting and internal elements are focused. Instead - the entire grid should be in a focused state as a single element, next tab should jump to the next control.

**Comment by kof** [comment]
moving that one on "after beta" project

**Comment by TrySound** [comment]
Seems like was fixed though navigation with arrows doesn't work
